Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/chan420/Chan420.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Kuroba - *chan browser https://github.com/Adamantcheese/Kuroba/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.github.adamantcheese.chan.core.site.sites.chan420;
+
+import androidx.annotation.Nullable;
+
+import com.github.adamantcheese.chan.core.model.Post;
+import com.github.adamantcheese.chan.core.model.orm.Board;
+import com.github.adamantcheese.chan.core.model.orm.Loadable;
+import com.github.adamantcheese.chan.core.site.Site;
+import com.github.adamantcheese.chan.core.site.SiteIcon;
+import com.github.adamantcheese.chan.core.site.common.CommonSite;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaActions;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaApi;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaCommentParser;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaEndpoints;
+import com.github.adamantcheese.chan.utils.Logger;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import okhttp3.HttpUrl;
+
+public class Chan420 extends CommonSite {
```

Comment:
You're working with an older version of the code, so before merging you'll need to add in ChunkDownloaderSiteProperties and the mediaHosts fields. For ChunkDownloaderSiteProperties, both should be false, as the byte count is wrong and no hash is sent. for mediaHosts you should use boards.420chan.org. Take a look at how those are implemented in the current code, using Sushichan and Chan4 as examples.

Reply:
commited the changes as requested derived from looking at examples.
tested based on current code (6c52470) and verified working.

i'll leave this open for review just in case.